### ISSUE TITLE
[Feature]: Add net6.0 TFM for Microsoft.Playwright.CLI

### ISSUE THREAD

**Issue by martincostello** [microsoft/playwright-dotnet#1668]
### Feature request

While targeting the .NET 6 pre-releases it is necessary to install both the .NET 5 and .NET 6 SDKs in a CI environment so that `Microsoft.Playwright.CLI` is able to run successfully. Otherwise it outputs an errors similar to this when `playwright install` is run:

```sh
    It was not possible to find any compatible framework version
    The framework 'Microsoft.NETCore.App', version '5.0.0' was not found.
      - The following frameworks were found:
          6.0.0-preview.7.21377.19 at [/home/runner/.dotnet/shared/Microsoft.NETCore.App]
    
    You can resolve the problem by installing the specified framework and/or SDK.
    
    The specified framework can be found at:
      - https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=5.0.0&arch=x64&rid=ubuntu.20.04-x64
```

This is fine as a workaround for now, but the tool should support running under the .NET 6 runtime once it is generally available in November.

**Comment by TanayParikh** [comment]
@martincostello's suggestion [here](https://github.com/dotnet/aspnetcore/pull/38414#discussion_r756970668) helped workaround this issue for now (Thanks!). Just leaving this comment as a reference for others who may encounter this issue.

**Comment by mxschmitt** [comment]
This should be fixed in 1.2.1 of the CLI.
